cluster/charts/rook/templates/clusterrolebinding.yaml

```diff
@@ -14,4 +14,25 @@ subjects:
 - kind: ServiceAccount
   name: rook-operator
   namespace: {{ .Release.Namespace }}
+{{- if .Values.pspEnable }}
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRoleBinding
```

Comment:
what's the difference between this cluster role binding and the existing one? Looks like the `rook-agent` service account was added. Should we just add that SA to the existing binding instead of create a new one?